courses/free-courses-ml.md

```diff
@@ -177,6 +184,12 @@
 * [SQL / MYSQL Malayalam Tutorial Full Course](https://www.youtube.com/playlist?list=PL1JrLEBAapUXMuDKwPVVzhlBge9fdxV51) - Yes Tech Media
 
 
+### Networking
+
+* [Computer Networks - S5](https://www.youtube.com/playlist?list=PLI74-7rtCb9BRUw6JzCm_wwLdQJc3jSwO) -  Eduline CSE Knowledge Sharing Platform
```

Comment:
could you switch the order of these two to make it alphabetical. Our linter is not checkingg malalayam.

Reply:
Resolved.